test/acceptance/features/steps/steps.py

```diff
@@ -276,3 +276,50 @@ def then_envFrom_contains(context, app_name, sbr_name1, sbr_name2):
 def then_envFrom_contains_intermediate_secret_name(context, intermediate_secret_name):
     assert context.application.get_deployment_with_intermediate_secret(
         intermediate_secret_name, wait=True, timeout=120) is not None, f"There is no deployment with intermediate secret {intermediate_secret_name}"
+
+
+# STEP
+@given(u'Backend infrastructure is running')
+def given_backend_infrastructure(context):
+    openshift = Openshift()
+    _ = openshift.oc_apply_yaml_file(os.path.join(os.getcwd(), "test/operator-hub/backend-csv.yaml"))
+    _ = openshift.oc_apply_yaml_file(os.path.join(os.getcwd(), "test/third-party-crds/backend-crd.yaml"))
+
+
+# STEP
+@given(u'Backend CR "{cr_name}" is applied')
+def create_backend_cr(context, cr_name):
+    openshift = Openshift()
+    yaml = context.text
+    output = openshift.oc_apply(yaml)
+    result = re.search(rf'.*{cr_name}.*(created|unchanged)', output)
+    result | should_not.be_none.desc("CRD {cr_name} Created")
+
+
+# STEP
+@given(u'Service Binding request "{sbr_name}" is applied')
+def create_sbr(context, sbr_name):
+    openshift = Openshift()
+    yaml = context.text
+    output = openshift.oc_apply(yaml)
+    result = re.search(rf'.*{sbr_name}.*(created|unchanged)', output)
+    result | should_not.be_none.desc("SBR {sbr_name} Created")
+
+
+# STEP
+@when(u'Backend status in "{backend_name}" is updated')
+def when_backend_status_updated(context, backend_name):
+    openshift = Openshift()
+    yaml = context.text
+    output = openshift.oc_apply(yaml)
+    result = re.search(rf'.*{backend_name}.*(created|unchanged|configured)', output)
+    result | should_not.be_none.desc("Backend {backend_name} updated")
+
+
+# STEP
+@then(u'Secret "{secret_name}" contains CustomReady custom env name with value true')
```

Comment:
`CustomReady` should be function param, value as well.

We should really decode the secret and look inside. See https://kubernetes.io/docs/concepts/configuration/secret/#decoding-a-secret

Reply:
Decoded the secret https://github.com/redhat-developer/service-binding-operator/pull/597/commits/2899fd0b882ae30ee3b19d149d9329703c878e0c 
I am keeping it as CustomReady for now because while mentioning the jsonpath with oc I really need the data name. I could not find a way where I can send a parameter to the jsonPath format with oc

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/597/commits/b904a2186b93b06ef06213f2a23139572628abdc